src/pint/observatory/__init__.py

```diff
@@ -59,6 +59,7 @@
 
 # The default BIPM to use if not explicitly specified
 # FIXME: this should be auto-detected by checking the index file to see what's available
+# ALSO: Why is the a feature of the Observatory class?  Seems like the wrong place for it.
```

Comment:
May be we can move this to the `pint` namespace or `pint.toa` namespace?

Reply:
I looked at this and in the end decided it was better where it was. The lowest level clock correction calculator is in the Observatory class. So I left it and deleted that comment.

What do you think about making the default auto-detect the most recent BIPM available?  I could add that.

Reply:
> What do you think about making the default auto-detect the most recent BIPM available?

That makes sense.